templates/packer-ansible.dockerfile

```
@@ -0,0 +1,16 @@
+FROM ubuntu:18.04
+
+ENV PATH=/root/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

```suggestion
ENV PATH=/root:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

maybe we should remove this trailing slash to be consistent with the rest of the folders in this line. although if the `/root` folder is not needed on the path then this line may be able to be removed entirely.